WalletWasabi.Fluent.Generators/NavigationMetaDataGenerator.cs

```diff
@@ -123,14 +126,34 @@ public partial class {{classSymbol.ToDisplayString(format)}}{{implementedInterfa
 
 			source.AppendLine($"\t\t{namedArgument.Key} = " +
 							  $"{(namedArgument.Value.Kind == TypedConstantKind.Array ? "new[] " : "")}" +
-							  $"{namedArgument.Value.ToCSharpString()}{(i < length - 1 ? "," : "")}");
+							  $"{namedArgument.Value.ToCSharpString()},");
 		}
 
-		source.Append(
-			"""
-				};
+		var isLocalized = attributeData.NamedArguments.FirstOrDefault(x => x.Key == "IsLocalized").Value.Value;
```

Comment:
NRE here. In case there is always at least one element with key "IsLocalized" then use First instead of FirstOrDefault

Reply:
There is no NRE. But I can see if I can make more explicit